docs/fast_data/configuration/single_view_creator.md

```diff
@@ -570,6 +572,21 @@ module.exports = (logger, projection) => {
 }
 ```
 
+## Single View Retry
```

Comment:
@ianFar96 aggiungiamo una tabella con il tipo di errore che viene generato e una spiegazione di quando viene generato? E se quell'errore è gestito dal meccanismo di sv-retry? Potresti anche creare un altro paragrafo, non lo metterei necessariamente nel sv-retry, alla fine la generazione dell'errore c'è a prescindere. Eventualmente è da mettere vicino a dove parliamo della collezione di sv-error (se ne parliamo) e così puoi aggiungere l'informazione che viene generata in Console nella sezione mongoDB CRUD

Reply:
Fatto, fatemi sapere @flower-of-the-bridges @TheRusCompass